mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/SimpleXMLTest.java

```diff
@@ -0,0 +1,81 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.SimpleXMLPage;
+
+public class SimpleXMLTest extends BasePageWebApiTest {
+	
+	private static SimpleXMLPage simpleXMLPage ;
+	private static Document simpleXMLPageDocument;
+
+	@BeforeClass
+	public static void setup(){
+		simpleXMLPage = new SimpleXMLPage();
+		DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+		BFLogger.logDebug("Reading " + simpleXMLPage.getEndpoint());
+		try {
+			DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+			simpleXMLPageDocument = dBuilder.parse(simpleXMLPage.getXMLDocument().asInputStream());
+			simpleXMLPageDocument.normalizeDocument();
+		} catch (ParserConfigurationException e) {
+			e.printStackTrace();
```

Comment:
Consider some descriptive messages being logged in catch clauses instead of just printing stack traces...

Reply:
fixed